lib/src/Base/Graph/openturns/Drawable.hxx

```diff
@@ -83,6 +83,9 @@ public:
   void setColor(const String & color);
   String getEdgeColor() const;
 
+  /** Explicit color validation flag accessor */
+  virtual Bool isColorExplicitlySet() const;
```

Comment:
here too, move it to protected (or can even be dropped and called from getImplementation())